CDCgov/prime-reportstream
# DE - No Blanks for ORC-12-2 / ORC-12-3

This PR documents the changes to the settings file for DE where ORC-12-2 and ORC-12-3 will substitute blank input for "Dr. Gregory" "Warner". At the same time, this PR adds the changes to the local org file.

#### Note
Changes are already in prod and staging

Test Steps:
1. Create fake data for DE
2. Edit fake data by blanking out Ordering Provider Name/Lastname
3. Run docker
4. reloadsettings
5. Send fake file to api
6. verify output has "Dr. Gregory" for ORC-12-2 and "Warner" for ORC-12-3

## Changes
- Local org file changes to the DE receiver; used replaceValueAwithB
- Two YML files to keep record of the changes

### Testing



## Linked Issues
- Fixes #6786
